maventa/types/MaventaInvoice.ts

```diff
@@ -0,0 +1,28 @@
+import { MaventaAction } from "./MaventaAction";
+import { MaventaRecipient } from "./MaventaRecipient";
+import { MaventaSender } from "./MaventaSender";
+
+
+interface MaventaInvoice {
+  id: string;
+  status: string;
+  reference: string | null;
+  number: string;
+  sender: MaventaSender;
+  recipient: MaventaRecipient;
+  received_at?: string;
+  created_at: string;
+  date: string;
+  date_due: string;
+  source_format: string;
+  sum: number;
+  sum_tax: number;
+  currency: string;
+  destination: string | null;
+  comment: string | null;
+  files: Array<File>;
+  actions: Array<MaventaAction>;
+  revision: Record<string, unknown>;
```

Comment:
This might be simpler to define as an interface like this:

```
interface MaventaRevisionObject {
  [key: string]: unknown;
}
```

Why is it `unknown`? We don't know what it is?